doc/src/records/dhwsys.md

```diff
@@ -186,21 +312,71 @@ Day waste scaling factor.
   ----------- ----------------- ------------------------------ -------------- -----------------
      --        $\ge$ 0             1                               No            constant
 
+**wsUnkDayWasteF=*float***
 
+Unknown relative water draw for day of waste scheme.
```

Comment:
The CNRECS comment isn't very clear here. Maybe @chipbarnaby can come up with better description text?